controller/state/statedb.go

```diff
@@ -144,6 +148,19 @@ func newStateDBWithNotify(ctx context.Context, dbConn DBConnector, migrator Migr
 		log.Infow("recovered scheduled tasks", "task_count", count)
 	}
 
+	storeLS := storeLS(st.Store(context.Background()))
+	host, err := NewHost(identity, addrs)
+	if err != nil {
+		return nil, err
+	}
+
+	b := dbDS("legs_data", st.db())
+	pub, err := dtsync.NewPublisher(host, b, storeLS, "/dealbot/v1.0.0")
```

Comment:
Can we make the topic configurable? 

Side note: should this be Pando topic or is the expectation that Pando producers can also specify topic?

Reply:
happy to use a pando topic if there is one